### PR TITLE
ALBS-289 Add information about built modules

### PR DESCRIPTION
**Pull request by maccelf**
We need to expose information about built modules for the build.

### CODE REVIEW

File: src/pages/Build.vue [hunk 1]
```diff
@@ -848,6 +876,9 @@ export default defineComponent({
         return false
       }
       return true
+    },
+    moduleEVR(module) {
```
Comment (anfimovdm) [inline src/pages/Build.vue:880]: it should be NVSCA, name-version-context-arch

**Review by Korulag — APPROVED** [review]

**Review by anfimovdm — CHANGES_REQUESTED** [review]

File: src/pages/Build.vue [hunk 2]
```diff
@@ -51,6 +51,18 @@
                 </tr>
               </tbody>
             </table>
+            <q-expansion-item label="Built modules" expand-separator
+                            icon="lock" v-if="rpm_module && Object.keys(rpm_module).length !== 0" class="q-pt-lg">
```
[the remark this answers is not in the document]
Comment (maccelf) [reply src/pages/Build.vue]: Done

File: src/pages/Build.vue [hunk 3]
```diff
@@ -108,6 +120,18 @@
...
```
[the remark this answers is not in the document]
Comment (maccelf) [reply src/pages/Build.vue]: Done

File: src/pages/Build.vue (hunk 1 above)
Comment (maccelf) [reply src/pages/Build.vue]: Done

**Review by AvoidMe — APPROVED** [review]

**Review by anfimovdm — APPROVED** [review]

**Pull request merged**